WooCommerce/Classes/Authentication/Epilogue/AccountHeaderView.swift

```diff
@@ -101,8 +101,8 @@ private extension AccountHeaderView {
     func setupHelpButton() {
         helpButton.setTitle(Strings.helpButtonTitle, for: .normal)
         helpButton.setTitle(Strings.helpButtonTitle, for: .highlighted)
-        helpButton.setTitleColor(.primary, for: .normal)
-        helpButton.setTitleColor(.textSubtle, for: .highlighted)
+        helpButton.setTitleColor(.accent, for: .normal)
+        helpButton.setTitleColor(.accent, for: .highlighted)
```

Comment:
nit: since the button is `system` type (set in the xib), the highlighted color is probably not necessary? I believe the highlighted state color only works when the button type is `custom`.

right now, the highlighted state looks like (which looks good to me):

normal state | highlighted state
-- | --
![Simulator Screen Shot - iPhone 8 Plus - 2021-01-06 at 17 58 19](https://user-images.githubusercontent.com/1945542/103756086-d0bfd680-5049-11eb-9164-e283624f2f72.png) | ![Simulator Screen Shot - iPhone 8 Plus - 2021-01-06 at 17 58 24](https://user-images.githubusercontent.com/1945542/103755992-b128ae00-5049-11eb-9a50-643f01a7911a.png)

Reply:
Good catch. I'll remove the highlighted setter entirely for the helpButton. Thanks!

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3428/commits/aa065f1583ffeed9b4607a2b4acd1c5a042df431

I also removed the setTitle for highlighted for the same reason